builtin/providers/aws/resource_aws_security_group_rule.go

```diff
@@ -263,15 +267,32 @@ func findResourceSecurityGroup(conn *ec2.EC2, id string) (*ec2.SecurityGroup, er
 	return resp.SecurityGroups[0], nil
 }
 
+// byUserIDAndGroup implements sort.Interface for []*ec2.UserIDGroupPairs based on
+// UserID and then the GroupID or GroupName field (only one should be set).
```

Comment:
ah, I removed `UserID` since we don't set it (we ignore it completely) . I changed the method name and docs in 640836e